eosdis_store/dmrpp.py

```diff
@@ -210,8 +214,15 @@ def array_to_zarr(node, dims, prefix=''):
     # NOTE - this is null in test file
     zarray['fill_value'] = zattrs.get('_FillValue')
 
-    #if node.attrib['name'] == 'time':
-    #    set_trace()
+    # Automatic scale factor and offset filter.  Not yet reliable
```

Comment:
Not yet reliable?

Reply:
Do you have a ticket to make this work? If so you should add it to the comment.

Reply:
It only works in some special cases when variable data types don't change. I'll clarify and add a ticket and ticket number